web-client/netlify.toml

```diff
@@ -3,3 +3,9 @@
   to = "/index.html"
   status = 200
 
+[[headers]]
+  for = "/*"
+  [[headers.values]]
+    X-Frame-Options = "DENY"
```

Comment:
This should be only a workaround as it's deprecated and should be handled in the csp with `frame-src` set to `self` or the Tauri host once we use it inside Tauri.

If we add it to the the CSP in production I would keep the `x-frame` header it until we enforce the CSP in production.